src/CroctProvider.tsx

```diff
@@ -2,7 +2,9 @@ import {createContext, FunctionComponent, ReactElement, useContext, useEffect, u
 import {Configuration, Plug} from '@croct/plug';
 import {croct} from './ssr-polyfills';
 
-export type CroctProviderProps = Configuration & Required<Pick<Configuration, 'appId'>>;
+export type CroctProviderProps = Configuration & Required<Pick<Configuration, 'appId'>> & {
+    children?: ReactElement,
```

Comment:
```suggestion
    children?: ReactNode,
```

Reply:
Or you can use the generic `PropsWithChildren` when using a `FunctionComponent`.

```ts
// react/index.d.ts
type PropsWithChildren<P> = P & { children?: ReactNode };
```

eg: 

```tsx
export const  CroctProvider: FunctionComponent<PropsWithChildren<CroctProviderProps>> = (...) => {};
```

> I agree with `ReactNode` and maybe you don't need to use `FunctionComponent`.

Reply:
or just:

``ts
type CroctProviderProps = PropsWithChildren<...>;
``